client/my-sites/checklist/style.scss

```diff
@@ -0,0 +1,214 @@
+.checklist-header {
```

Comment:
Note that we can add a `/** @format */` header, for prettier autoformatting now.